scripts/docs-content-link-rewrites/helpers/get-mdx-links-to-rewrite.ts

```diff
@@ -40,6 +43,22 @@ const getMdxLinksToRewrite = async ({
 		}
 
 		const fileContent = fs.readFileSync(filePath, 'utf-8')
+
+		/**
+		 * Ignore files that have a different `source` than the current repo. This
+		 * property indicates that the content is regularly copied over from the
+		 * source, and thus should not be modified.
+		 *
+		 * Example file in the `ptfe-releases` repository:
+		 * https://github.com/hashicorp/ptfe-releases/blob/ba4dc2af20cc9d0f46247c919be8b2ed90d1e6f8/website/docs/enterprise/registry/index.mdx?plain=1#L6
+		 */
+		const { data: frontmatter } = matter(fileContent)
+		const isSourcedFromDifferentRepo =
+			typeof frontmatter?.source === 'string' && frontmatter.source !== repo
+		if (isSourcedFromDifferentRepo) {
+			continue
+		}
```

Comment:
@thiskevinwang Is the existence of the `source` frontmatter property enough to know we should ignore the document, or do we need to do a check like this? ☝️

Reply:
Is the goal/intent to ignore a class of files managed via automation?

If so it _feels like_ simply checking for existence of `source` should suffice here.